src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -273,12 +273,47 @@ private void testRemindActions() {
         homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_REMINDERSSENT);
         homePage.goToPreviousPage(InstructorHomePage.class);
 
-        ______TS("remind action: CLOSED feedback session");
 
-        homePage.verifyUnclickable(homePage.getRemindLink(feedbackSessionClosed.getCourseId(),
```

Comment:
Can we use this example to `verifyUnclickable` for the remind links of awaiting and published sessions?